tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_search_bug.py

```diff
@@ -0,0 +1,24 @@
+from gaiatest import GaiaTestCase
+
+from gaiatest.apps.bzlite.app import BugzillaLiteStage
+
+class TestSearchBug(GaiaTestCase):
+     def setUp(self):
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.connect_to_local_area_network() # <= No detail about how to connect
+        name = 'Bugzilla Lite Stage'
+        url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+        self.apps.install_app(name, url) # <= No detail about how to install
+
+     def test_search_bug(self):
+        bugzilla_lite = BugzillaLiteStage(self.marionette)
+        bugzilla_lite.launch()
+
+        bugzilla_lite.dismiss_tooltip()
+        bugzilla_lite.dismiss_content()
+
+        bugzilla_lite.login(self.testvars['bugzillaStage']['user'], self.testvars['bugzillaStage']['password'])
+
+        bugzilla_lite.search(str(self.testvars['bugs']['bug2']))
```

Comment:
Using a variable in testvars is needed:
* either when you don't want to make the data public (like a password)
* or when the data to vary from a user to another (like a phone number to call, depending on your country or set up)

Here, I don't think the bug number is private or will vary from a user to another. We can hardcode the bug number in the test, then.